terraform/Mo/rancher-master/rancher-master.tf

```diff
@@ -1,10 +1,9 @@
 resource "openstack_compute_instance_v2" "rancherMaster" {
   name            = "rancherMaster"
   flavor_name     = "g1.large_lowmem"
-  key_pair        = "<YOURKEYPAIR>"
+  key_pair        = "rancher-master"
```

Comment:
This breaks the idea of the example repo. The user should insert the ID of his SSH keypair. Description of that is in the base folders README